west.yml

```diff
@@ -38,7 +38,7 @@ manifest:
     - name: zephyr
       repo-path: fw-nrfconnect-zephyr
       west-commands: scripts/west-commands.yml
-      revision: pull/198/head
```

Comment:
maybe we could also fix this reference in the same PR?

Reply:
I need to rebase the upmerge PR anyway, so it doesn't really make a difference